torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3987,7 +3987,7 @@ def sample_inputs(op_info: OpInfo, device: torch.device, dtype: torch.dtype,
         ]
 
         sample_inputs = []
-        for shapes, indexing in itertools.product(test_cases, {'ij'}):
+        for shapes, indexing in itertools.product(test_cases, {'ij', 'xy'}):
```

Comment:
Cool extension

Reply:
What about existing meshgrid testing? Sample inputs comparing with a NumPy reference are great, but typically sample inputs are minimal (because they're used in so many tests). Are there more thorough "forward only" meshgrid tests that compare against a NumPy reference and test edge cases? Is there a test that checks for invalid inputs, too?

Reply:
😉 

Sometimes it's nice to look into the future to anticipate what earlier parts of the chain should look like.